### PR TITLE
fix: build config

### PR DESCRIPTION
**Pull request by marianfoo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "dependencies": [
       "ui5-middleware-livereload",
       "ui5-tooling-transpile",
-      "ui5-tooling-modules"
+      "ui5-tooling-modules",
+      "firebase"
```
Comment (petermuessig) [inline package.json]: shouldn't be necessary - the ui5-tooling-modules use the native `require.resolve` lookup

File: ui5.yaml [hunk 2]
```diff
@@ -17,10 +17,10 @@ builder:
           afterTask: replaceVersion
           configuration: &transpileConfig
               removeConsoleStatements: true
-              transpileAsync: true
+              transpileAsync: false
```
Comment (petermuessig) [inline ui5.yaml]: should not be necessary

File: ui5.yaml [hunk 3]
```diff
@@ -17,10 +17,10 @@ builder:
...
               transpileTypeScript: true
         - name: ui5-tooling-modules-task
-          afterTask: replaceVersion
+          afterTask: ui5-tooling-transpile-task
```
Comment (petermuessig) [inline ui5.yaml:23]: the order of the plugin is important - transpiling must happen first. but as said - I can check tomorrow.

File: package.json (hunk 1 above)
Comment (marianfoo) [reply package.json]: thought so

File: ui5.yaml (hunk 3 above)
Comment (marianfoo) [reply ui5.yaml:23]: yeah, i think that´s the important part

Comment (marianfoo) [reply ui5.yaml:23]: tried again, thats the only change needed

**Comment by cpf-hse** [comment]
@marianfoo thank you for checking this.
I tried it and the only thing that is needed is updating the sequence in the builder tasks:
`        - name: ui5-tooling-modules-task
          afterTask: ui5-tooling-transpile-task`

Closing this PR

**Pull request closed without merging**